app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/vHeaderImage"
+        android:layout_width="match_parent"
+        android:layout_height="204dp"
+        android:layout_marginBottom="68dp"
+        android:scaleType="fitXY"
+        android:background="@drawable/login_cover"
+        android:src="@drawable/wolox_colors_gradient"
+        app:layout_constraintTop_toTopOf="parent"></ImageView>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@+id/vLoginButton"
+        app:layout_constraintTop_toTopOf="@id/vHeaderImage">
+
+        <ImageView
+            android:id="@+id/vWoloxLogoImage"
+            android:layout_width="match_parent"
+            android:layout_height="145dp"
+            app:srcCompat="@drawable/wolox_logo_large"></ImageView>
+    </RelativeLayout>
+
+    <EditText
+        android:id="@+id/vFirstNameInput"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="90dp"
+        android:layout_marginRight="16dp"
```

Comment:
add dimen to resources. For the 16dp dimens, you can use the once that are already defined in the dimens file